news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,139 @@
+date: 2024-09-17 12:00 +0000
+---
+
+The time for change is finally here!
+
+![](/wiki/shared/news/banners/pp-sr-osu.jpg)
+
+There are changes to all game modes this time around in an effort to improve the difficulty and performance calculations. This news post will display what has changed in a format that should be understandable to you, the player!
+
+If you find yourself scratching your head when reading, consider consulting the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) wiki articles to gain a greater understanding of the topics.
+
+## Release schedule
+
+<!-- todo -->
+
+## osu!catch performance points committee
+
+Since the last performance points post, osu!catch has now formed its own Performance Points Committee to help out with osu!catch related changes. You can find out more about the various committees on their [wiki page](/wiki/People/Performance_Points_Committee).
+
+---
+
+## osu! changes
+
+As a reminder of the various core values in the difficulty calculation algorithm:
+
+- **Aim**: how difficult it is to consistently hit notes back-to-back (affected by approach rate, jumps, Hidden, Hard Rock and more)
+- **Speed**: how quickly new elements are presented to the player (affected by note density, BPM, Double Time and Half Time)
+- **Accuracy**: how difficult it is to stay rhythmically on time (affected by Hidden, Hard Rock, Flashlight and more)
+- **Strain**: the difficulty of enduring intense or technically demanding patterns often and for a long time (affected by streams, fast jumps and more)
+
+### Combo scaling removal
+
+A [change](https://github.com/ppy/osu/pull/16280) submitted by [moonpoint](https://osu.ppy.sh/users/9558549) with improvements made by [KermitNuggies](https://osu.ppy.sh/users/33452559) has been created in order to remove the combo scaling factor of a score on *both* aim and speed.
+
+This change means a 1 miss score with a max combo of 500 compared to a 1 miss score on the same beatmap with a max combo of 700 will both be punished equally. The miss penalty was also rewritten in order to accomodate these changes - misses have generally been made harsher, with the initial miss having a harsher penalty in order to differentiate between FCs and non-FCs.
+
+The miss penalty scaling is now based on the amount of difficult strains instead of object count. As the penalty is based upon the amount of difficult strains, this means longer maps with a lot of "filler" sections will be punished harder than longer beatmaps which are consistently difficult even if both beatmaps have similar object counts.
+
+There were concerns raised about consistency being less important with these changes, however this isn't an issue. Using combo as a metric doesn't make much sense as the performance calculator does not know what sections of a beatmap that a score got misses on - as a result, we model any misses as being on the most difficult part.
```

Comment:
First sentence reads really bad, stops halfway through, weird.

Reply:
I've tried to improve it by connecting the two, but the sentence might be *too* long now. Open to suggestions on rewording because I can't really think of much.